dcos/marathon.py

```diff
@@ -130,6 +130,8 @@ class Client(object):
     """
 
     def __init__(self, marathon_url, timeout=http.DEFAULT_TIMEOUT):
+        if not marathon_url.endswith('/'):
+            marathon_url += '/'
         self._base_url = marathon_url
```

Comment:
How about `os.path.join(marathon_url, '')`? can we move this to `_get_marathon_url`? @vespian